src/tests/test_parse.py

```diff
@@ -88,6 +88,10 @@ def test_parse_shovel_operator():
         "dest      =<< SELECT * FROM work",
         "dest =          << SELECT * FROM work",
         "dest=<< SELECT * FROM work",
+        "dest=<<SELECT * FROM work",
```

Comment:
we're missing the test cases from the issue:

https://github.com/ploomber/jupysql/issues/610

please make sure we have all of them